.github/workflows/ci.yml

```diff
@@ -36,12 +36,21 @@ jobs:
       fail-fast: false
       matrix:
         include:
+
+          # latest stable build
           - sublime-channel: stable
             sublime-build: 4126
             optional: true
+
+          # latest dev build
           - sublime-channel: dev
-            sublime-build: 4125
+            sublime-build: 4127
             optional: false
+
+          # first build with the latest changes to the syntax engine in core
+          - sublime-channel: dev
+            sublime-build: 4095
```

Comment:
What is the motivation for this? I personally don't think we need to spend time trying to keep compatibility with older builds, even seeing if it is compatible isn't interesting for me :)

Reply:
Ok

Reply:
I thought it might make sense to check against the documented first build with the latest changes to the syntax engine in core.

So if SublimeHQ adds any changes to `master` with undocumented changes to the syntax engine that would break that test.

But maybe this is the wrong approach. I removed that part for now.